pkg/cmd/serviceaccount/accountcmdutil/account_util.go

```diff
@@ -0,0 +1,7 @@
+package accountcmdutil
```

Comment:
I would call it `svcaccountcmdutil` personally as account could be associated with user accounts etc by people reading the code.